modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/idp/mgt/ResidentIDPConfigsTestCase.java

```diff
@@ -0,0 +1,262 @@
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OAuth2RevokeEPUrl"),
+                "https://localhost:9853/oauth2/revoke",
+                "Expected Token Revocation Endpoint URL is not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OIDCLogoutEPUrl"),
+                "https://localhost:9853/oidc/logout",
+                "Expected Logout Endpoint URL not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OAuth2AuthzEPUrl"),
+                "https://localhost:9853/oauth2/authorize",
+                "Expected Authorization Endpoint URL is not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OAuth2DCREPUrl"),
+                "https://localhost:9853/t/wso2.com/api/identity/oauth2/dcr/v1.1/register",
+                "Expected Dynamic Client Registration Endpoint URL is not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OIDCDiscoveryEPUrl"),
+                "https://localhost:9853/t/wso2.com/oauth2/oidcdiscovery",
+                "Expected Dynamic Client Registration Endpoint URL is not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OAuth2JWKSPage"),
+                "https://localhost:9853/t/wso2.com/oauth2/jwks",
+                "Expected is not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OAuth2TokenEPUrl"),
+                "https://localhost:9853/oauth2/token",
+                "Expected Token Endpoint URL is not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OAuth2UserInfoEPUrl"),
+                "https://localhost:9853/oauth2/userinfo",
+                "Expected User Info Endpoint URL is not found in the tenant mode");
+
+        //Extract provisioning configurations for tenant.
+        ProvisioningConnectorConfig[] provisioningConfigsTenant = residentIdP.getProvisioningConnectorConfigs();
+
+        Map<String, String> provisioningConfigTenantMap = new HashMap<String, String>();
+        for (ProvisioningConnectorConfig config : provisioningConfigsTenant) {
+            for (Property property: config.getProvisioningProperties()) {
+                provisioningConfigTenantMap.put(property.getName(), property.getValue());
+            }
+        }
+
+        Assert.assertEquals(provisioningConfigTenantMap.get("scimGroupEndpoint"),
+                "https://localhost:9853/wso2/scim/Groups",
+                "Expected SCIM Group Endpoint is not found in the tenant mode");
+        Assert.assertEquals(provisioningConfigTenantMap.get("scimUserEndpoint"),
+                "https://localhost:9853/wso2/scim/Users",
+                "Expected SCIM User Endpoint is not found in the tenant mode");
+        Assert.assertEquals(provisioningConfigTenantMap.get("scim2UserEndpoint"),
+                "https://localhost:9853/t/wso2.com/wso2/scim2/Users",
```

Comment:
Is this URL correct? This should be ```https://localhost:9853/t/wso2.com/scim2/Users```?

Reply:
This is correct. for SCIM 2.0 User Endpoint URL is `https://localhost:9443/t/wso2.com/wso2/scim2/Users`

Reply:
I think its a bug :)
we need to fix it and change the tests as well

Reply:
Bug was fixed in PR: https://github.com/wso2/carbon-identity-framework/pull/2946 and tests were changed in https://github.com/wso2/product-is/pull/8370/commits/f196fdb5ff9850329b87aaffb483f2d68783cb04